src/devices/machine/myb3k_kbd.h

```diff
@@ -30,8 +30,12 @@
 #define MCFG_MYB3K_KEYBOARD_CB(cb)          myb3k_keyboard_device::set_keyboard_callback(*device, (MYB3K_KBD_CB_##cb));
 
 DECLARE_DEVICE_TYPE(MYB3K_KEYBOARD, myb3k_keyboard_device)
+DECLARE_DEVICE_TYPE(JB3000_KEYBOARD, jb3000_keyboard_device)
+DECLARE_DEVICE_TYPE(STEPONE_KEYBOARD, stepone_keyboard_device)
 
 INPUT_PORTS_EXTERN( myb3k_keyboard );
+INPUT_PORTS_EXTERN( jb3000_keyboard );
+INPUT_PORTS_EXTERN( stepone_keyboard );
```

Comment:
You shouldn't need to declare these here since the device instantiates them - they can be `static` or in an anonymous namespace in the source file.